CRM/Core/Config.php

```diff
@@ -388,7 +388,7 @@ public static function clearTempTables($timeInterval = FALSE): void {
       // TODO: Circa v5.60+, consider a more precise cleanup. Discussion: https://github.com/civicrm/civicrm-core/pull/24538
       // A separate process will reap the UserJobs but here the goal is just not to delete them during cache clearing
       // if they are still referenced.
-      if (!CRM_Core_DAO::executeQuery("SELECT count(*) FROM civicrm_user_job WHERE metadata LIKE '%" . $tableDAO->tableName . "%'")) {
+      if (!CRM_Core_DAO::singleValueQuery("SELECT count(*) FROM civicrm_user_job WHERE metadata LIKE '%" . $tableDAO->tableName . "%'")) {
```

Comment:
$tableDAO is being used here outside the loop. So it seems like multiple bugs here since this would only check the last one. I think this should be inside the loop:

```php
while ($tableDAO->fetch()) {
  if (!CRM_Core_DAO::singleValueQuery("SELECT count(*) FROM civicrm_user_job WHERE metadata LIKE '%" . $tableDAO->tableName . "%'")) {
    $tables[] = $tableDAO->tableName;
  }
}
```

Reply:
@demeritcowboy good catch - I update the PR